build.yml

```diff
@@ -206,7 +206,24 @@ targets:
     steps:
       # - $: "rm -f go.mod go.sum"
       # - $: "go mod init ={mod}"
-      - $: "go mod tidy"
+      - if: "!exists('web/play/node_modules')"
+        then:
+          - $: cd web/play/ && yarn install
+      - if: "!exists('internal/app/pimo/client/')"
+        then:
+          - $: cd internal/app/pimo/ && mkdir client && neon build-web
+          - $: "go mod tidy"
+          - $: "echo > web/play/node_modules/go.mod"
+          - if: "!exists('go.mod')"
+            then:
+              - $: "go mod init ={mod}"
```

Comment:
The `go mod tidy` is important, and is the main goal of the refresh target so it should not be removed.
Additionnaly, you should never call neon directly, but use the `depends` property.

maybe `yarn install` should have it's own target ?

Reply:
The rule build-web should 
- yarn install, if necessary (node_modules is empty except for go.mod)
- yarn build, always
- cp -r web/play/build/* internal/app/pimo/client, always